src/Filters/Product.jsx

```diff
@@ -0,0 +1,66 @@
+import AggregationBranch from './AggregationBranch'
+import CollapsibleFilter from './CollapsibleFilter'
+import { connect } from 'react-redux'
+import MoreOrLess from './MoreOrLess'
+import React from 'react'
+import { SLUG_SEPARATOR } from '../constants'
+import { sortSelThenCount } from './utils'
+
+export class Product extends React.Component {
+  constructor( props ) {
+    super( props )
+
+    this._onBucket = this._onBucket.bind( this )
+  }
+
+  render() {
+    const listComponentProps = {
+      fieldName: 'product'
+    }
+
+    return (
+      <CollapsibleFilter title="Product / sub-product"
+                         desc="The type of product and sub-product the consumer identified in the complaint"
+                         showChildren={this.props.showChildren}
+                         className="aggregation">
+        <a href="http://files.consumerfinance.gov/f/documents/201704_cfpb_Summary_of_Product_and_Sub-product_Changes.pdf" target="_blank">Recent changes to products and sub-products</a>
```

Comment:
It looks like this aggregation needs a slightly different style and additional spacing to match the comp. That said, that would not hold up merging this but would require adding a smaller clean-up task for a revisit.